logdna/client_test.go

```diff
@@ -0,0 +1,57 @@
+package logdna
+
+import (
+	"errors"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var url = "https://api.logdna.com"
+
+func TestClient_View(t *testing.T) {
+	payload := ViewPayload{Name: "test", Query: "test"}
+	client := Client{ServiceKey: servicekey, HTTPClient: &http.Client{Timeout: 15 * time.Second}}
```

Comment:
for testing http requests, I'd expect this to be using the [httptest](https://golang.org/pkg/net/http/httptest/) package, from which you can get a `http.Client` instance for testing. these tests should likely _not_ be hitting the actual api endpoint - they should hit a test server where you provide the response. the goal would be to test the behavior of `client.CreateView` in isolation for things like response handling and error conditions.